models/user.go

```diff
@@ -4,7 +4,7 @@ var PathUsers = "/users"
 
 //
 type UserBody struct {
-	Localation      string `json:"localation,omitempty"`
+	Location      string `json:"location,omitempty"`
```

Comment:
Just do be annoying, can we fix the indents :)
You should actually be able to just run `go fmt ./...`

Reply:
@wrighbr I think your VSCode linter should do this on every save, maybe run the setup script again ?

Reply:
all good I updated it using codespell to fix all my great spelling mistakes